public_html/formCreator/index.php

```diff
@@ -167,6 +171,8 @@
 			$engine->openDB->escape($form['formContainer']),
 			$engine->openDB->escape($form['formProduction']),
 			$engine->openDB->escape($form['formMetadata']),
+			$engine->openDB->escape($form['exportPublic']),       // export public
+			$engine->openDB->escape($form['exportOAI']),          // export OAI
```

Comment:
These comments don't serve any purpose and should be removed. They don't add anything that reading the code doesn't provide, and provide something else that needs to be maintained.